pkg/validation/app.go

```diff
@@ -47,6 +48,11 @@ func (v *Validator) ValidateApp(ctx context.Context, app v1alpha1.App) (bool, er
 		return false, microerror.Mask(err)
 	}
 
+	err = v.validateMetadataConstraint(ctx, app)
```

Comment:
```suggestion
	err = v.validateMetadataConstraints(ctx, app)
```

Can these methods be in alpha order?